WordPress/Classes/Services/ReaderSearchSuggestionService.swift

```diff
@@ -0,0 +1,83 @@
+import Foundation
+
+///
+///
```

Comment:
Oopsie.  I forgot to go back and document. Those were placeholders to remind me.